app/models/user.rb

```diff
@@ -46,6 +46,16 @@ def last_week_tasks(date)
     project.tasks.where(updated_at: (date.beginning_of_week..date.end_of_week))
   end
 
+  def self.mentees_missing_on_website
+    mentees_missing = User.mentee.where("last_activity_at is null or last_activity_at >= (?)", 15.days.ago)
```

Comment:
@anjali-sharma just noticed - it's a scope so you don't need to use `User` just `mentee.where...` is enough.

Also let's not store results in variables but just connect where and where.not like `mentee.where(...).where.not(..)`
same in mentors